docs/swagger.yaml

```diff
@@ -654,117 +841,95 @@ definitions:
       description:
         type: "string"
         description: "This field is visible to the end user in-app"
```

Comment:
This is also concatinated wit agreementName

Reply:
Needs explicit description of how concatenation works. Which fields, no whitespace added, etc.